client/src/app/sidebar/sidebar.component.css

```diff
@@ -0,0 +1,112 @@
+html, body {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    overflow: hidden; 
+  }
+  
+  .sidebar {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100px;
+    background-color: blue;
+    height: 100vh;
+    overflow-y: auto;
+  }
+  
+  #toggleSidebar {
+    position: absolute;
+    top: 20px; 
+    left: 150px; 
+    background: none;
+    border: none;
+    cursor: pointer;
+    z-index: 1; 
+  }
+  
+  .hamburger-menu {
+    display: block;
+    width: 30px; 
+    height: 20px;
+    position: relative;
+  }
+  
+  .hamburger-menu span {
```

Comment:
Just a note, if necessary prefix your CSS classes with the name of the component to avoid CSS collision.

You can also configure the encapsulation of the component.

Read more here -> https://www.pluralsight.com/guides/css-encapsulation-in-angular

Reply:
done